### ISSUE TITLE
Why are instruction tokens included in loss calculation during training?

### ISSUE THREAD

**Issue by zero2vec** [MichalZawalski/embodied-CoT#32]
I noticed that in the current implementation of `RLDSBatchTransform.__call__()` in `prismatic/vla/datasets/datasets.py`, the labels are set to include all tokens (instruction + reasoning + action), which means the loss is computed over the entire sequence including the user instruction.

This differs from the original OpenVLA implementation, where instruction tokens are masked out using `IGNORE_INDEX`.

## Current Implementation (Embodied-CoT)

```python
# prismatic/vla/datasets/datasets.py, line 97-111
input_ids = self.base_tokenizer(prompt_builder.get_prompt(), add_special_tokens=True).input_ids
labels = list(input_ids)  # Labels include all tokens

input_ids, labels = torch.tensor(input_ids), torch.tensor(labels)
pixel_values = self.image_transform(img)

if not self.predict_stop_token:
    labels[-1] = IGNORE_INDEX

return dict(pixel_values=pixel_values, input_ids=input_ids, labels=labels, dataset_name=dataset_name)
```

## Original OpenVLA Implementation

In the original OpenVLA repository, the instruction portion is explicitly masked:

```python
input_ids = self.base_tokenizer(prompt_builder.get_prompt(), add_special_tokens=True).input_ids
labels = list(input_ids)

# [CRITICAL] We do not want to take the loss for anything but the predicted action tokens!
labels[: -(len(action) + 1)] = IGNORE_INDEX
if not self.predict_stop_token:
    labels[-1] = IGNORE_INDEX
```

I'm curious about the design decision here. Is there a specific reason why Embodied-CoT computes loss over the entire sequence including the instruction tokens? 

Thank you for your work on this project! I'm trying to understand the training methodology better.
